src/components/block/DonationProjectCard/DonationProjectCard.js

```diff
@@ -0,0 +1,28 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+import s from "./DonationProjectCard.module.css";
+
+const DonationProjectCard = ({ title, subtitle }) => (
+  <div className={s.container}>
+    <div className={s.bottombox}>
+      <div className={s.bar} />
+      <div>
+        <div className={s.title}>{title}</div>
+        <div className={s.subtitle}>{subtitle}</div>
+      </div>
+    </div>
+  </div>
+);
+
+DonationProjectCard.propTypes = {
+  title: PropTypes.string,
+  subtitle: PropTypes.string,
+};
+
+DonationProjectCard.defaultProps = {
+  title: PropTypes.string.isRequired,
+  subtitle: PropTypes.string.isRequired,
+};
```

Comment:
This isn't valid. `isRequired` goes on the type definitions in `propTypes`. [`defaultProps`](https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values) is where you provide a value to use when the property is not passed in.

Reply:
👍 in that case, I changed the default value to null if there is no title or subtitle.

Reply:
Thanks! I think you may have not pushed those changes up to GitHub yet, since I don't see them here.

I'd also probably suggest that we make the `title` required, which means that you don't need to pass in a default value, since you are required to pass in a value. I don't think this component would make any sense without the title, which is why I think we should probably make it required. I think it's probably also worthwhile making the subtitle required, even if I could kind of see the component working without one.